mmv1/products/dlp/DeidentifyTemplate.yaml

```diff
@@ -42,8 +42,19 @@ examples:
     primary_resource_id: 'basic'
     test_env_vars:
       project: :PROJECT_NAME
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'dlp_deidentify_template_with_template_id'
+    primary_resource_id: 'with_template_id'
+    vars:
+      name: 'tf-test'
```

Comment:
The more common way to do this is choose any name with a hyphen, eg. `my-template`. This will show up as `my-template` in the docs, and `tf-test-my-template%{random}` in the test. This also means you can get rid of the `test_vars_overrides` block.

Reply:
Thanks for pointing this out. I've updated this.